en/vespa8-release-notes.html

```diff
@@ -893,6 +927,90 @@ <h2 id="removed-metrics">Removed metrics</h2>
       <td><em>http.status.403.rate</em></td>
       <td>Use <em>http.status.4xx.rate</em> with dimension <em>statusCode</em>==403</td>
     </tr>
+    <tr>
+      <td><em>vds.visitor.allthreads.averagevisitorlifetime.sum.average</em></td>
```

Comment:
Perhaps use strikethrough on 'sum.average' to mark that the '.sum' part is also gone? Some of these metrics are also renamed, right?

Reply:
I have use strikethrough on all elements that will be removed.
That is .sum and .alldisks